src/algebra/continued_fractions/basic.lean

```diff
@@ -220,13 +220,13 @@ variables [has_one α] [has_zero α] [has_lt α]
 instance has_coe_to_simple_continued_fraction : has_coe (cf α) (scf α) :=
 by {unfold cf, apply_instance}
 
-@[simp, elim_cast]
+@[simp, norm_cast]
 lemma coe_to_simple_continued_fraction {c : cf α} : (↑c : scf α) = c.val := rfl
 
 /-- Lift a cf to a scf using the inclusion map. -/
 instance has_coe_to_generalized_continued_fraction : has_coe (cf α) (gcf α) := ⟨λ c, ↑(↑c : scf α)⟩
 
-@[simp, elim_cast]
+@[simp, norm_cast push]
```

Comment:
On my machine the classifier failed. Since then I may have fixed the problem, I just have to remove the labels and verify they are categorized correctly.